src/Struct/FindologicProduct.php

```diff
@@ -325,6 +325,50 @@ protected function setUrl(): void
         $this->url = $productUrl;
     }
 
+    protected function getTranslatedSeoPath(): ?string
+    {
+        $salesChannel = $this->salesChannelContext->getSalesChannel();
+        $seoUrlCollection = $this->product->getSeoUrls()->filterBySalesChannelId($salesChannel->getId());
+
+        /** @var SeoUrlEntity|null $seoUrlEntity */
+        $seoUrlEntity = $this->getTranslatedEntity($seoUrlCollection);
+
+        return $seoUrlEntity ? $seoUrlEntity->getSeoPathInfo() : $seoUrlEntity;
```

Comment:
Either a string or null is returned, right? Could be more clear when you explicitly return `null` here